audbackend/core/backend.py

```diff
@@ -399,6 +407,7 @@ def put_archive(
             verbose: show debug messages
 
         Raises:
+            BackendError: if an error is raised on the backend
             FileNotFoundError: if one or more files do not exist
```

Comment:
To highlight that we are talking about the argument for local files here:
```suggestion
            FileNotFoundError: if one or more ``files`` do not exist
```